src/DataSource/Http.hs

```diff
@@ -239,9 +259,12 @@ _getUser = do
     apiCall     <- asksHTTPNetworkLayer hnlApiCall
 
     let url = showURL UserInfoURL
-    let req = apiRequest cfg url
-
-    apiCall parseJSON req
+    res <- runEitherT $ do
```

Comment:
You can simplify if you want to use transformers here and remove `transformers-either`:
```
    res <- runExceptT $ do
        req <- ExceptT . pure $ apiRequest cfg url
        ExceptT $ apiCall parseJSON req
```

But I don't think we will require this.

Reply:
`res <- runExceptT $ (ExceptT . pure $ apiRequest cfg url) >>= ExceptT . apiCall parseJSON`

Reply:
Putting it all into a single line doesn't seem like a good idea because it is harder to read.

Reply:
Yes, I know. The reason it's there is because it has nice symmetry, the one that's relevant and tells you how you can get rid of `transformers-either` and where it says that we won't require this is one above.

Reply:
Sure, I'll make the change you suggested above.